.build/BuildHelper/Get-Version.ps1

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+    .SYNOPSIS
+    Gets the current version.
+
+    .DESCRIPTION
+    Run this from the /src/scripts folder.
+
+    .PARAMETER Template
+    Optional. Name of the template or module to package. Default = * (all).
+
+    .PARAMETER Build
+    Optional. Indicates whether the Build-Toolkit command should be executed first. Default = false.
+
+    .EXAMPLE
+    ./Package-Toolkit
+
+    Generates ZIP files for each template using an existing build.
+
+    .EXAMPLE
+    ./Package-Toolkit -Build
+
+    Builds the latest code and generates ZIP files for each template.
+#>
+function Get-Version {
```

Comment:
I know its just a build script but it feels like this should be split up to avoid confusion on what it's actually doing. Right now it can either get or update an existing version. Get should probably be it's own function and update or set should increment accordingly and use get to return the updated value.